core/setup.py

```diff
@@ -64,30 +64,28 @@
         # ----
         # These packages are major-version-0. Keep upper bounds on upcoming minor versions (which could have breaking changes)
         # and check compatibility / bump in each new minor version of dbt-core.
-        "colorama>=0.3.9,<0.5",
         "pathspec>=0.9,<0.12",
-        "isodate>=0.6,<0.7",
-        # ----
         "sqlparse>=0.2.3,<0.5",
         # ----
         # These are major-version-0 packages also maintained by dbt-labs. Accept patches.
         "dbt-extractor~=0.5.0",
         "minimal-snowplow-tracker~=0.0.2",
         "dbt-semantic-interfaces~=0.5.0a2",
+        "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git@copy-code-from-dbt-core#egg=dbt",
```

Comment:
This will be updated to just "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git#egg=dbt" once https://github.com/dbt-labs/dbt-common/pull/13 is merged